internal/views/feeder.go

```diff
@@ -80,11 +80,26 @@ func (r *root) enter(row int) (feeder, error) {
 		return nil, errNoData
 	}
 	r.enteredAt = row
-	return newTopic(r.cli, r.topics[row], r.width, r.height, r.flashMessage)
+	return newTopic(r.cli, r.topics[row + r.height * r.pg], r.width, r.height, r.flashMessage)
 }
 
-func (r *root) jump(_ int64) error                                   { return nil }
-func (r *root) search(_ string, _ func(int64, int64)) (int64, error) { return -1, nil }
+func (r *root) jump(_ int64) error { return nil }
+func (r *root) search(search string, _ func(int64, int64)) (int64, error) {
+
+	var pa = -1
+	var cur = int64(-1)
+	for i := range r.topics {
+		if r.topics[i] == search {
```

Comment:
The other views use strings.Contains so you can get a positive search on a sub string.  I suggest:

if strings.Contains(r.topics[i], search) 

instead so the behavior matches the other searches.